src/opinion_summarisation/models/train.py

```diff
@@ -0,0 +1,102 @@
+
+# Commented out IPython magic to ensure Python compatibility.
+import pandas as pd
+import torch
+import sklearn
+from transformers import DistilBertTokenizer
+from sklearn.model_selection import train_test_split
+
+import logging
+from simpletransformers.seq2seq import (
+    Seq2SeqModel,
+    Seq2SeqArgs,
+)
+
+logging.basicConfig(level=logging.INFO)
+transformers_logger = logging.getLogger("transformers")
+transformers_logger.setLevel(logging.WARNING)
+
+if torch.cuda.is_available():
+    device = "cuda:0"
+else:
+    device = "cpu"
+
+train_df = pd.read_csv('/content/drive/MyDrive/Colab Notebooks/BachelorsProject/euans_reviews.csv')
+
+test_df = pd.read_csv("/content/drive/MyDrive/Colab Notebooks/BachelorsProject/google_reviews.csv")
+
+euans_reviews = train_df.Text.values.tolist()
+
+import time
```

Comment:
imports on top -> split into a few functions -> clean up (whole file)

Reply:
This will be moved to a different script altogether.